frontend/packages/data-portal/e2e/singleRunFilters.test.ts

```diff
@@ -0,0 +1,344 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
+import { test } from '@playwright/test'
+import { FiltersPage } from 'e2e/pageObjects/filters/filtersPage'
+
+import { QueryParams } from 'app/constants/query'
+
+import { getApolloClient } from './apollo'
+import { E2E_CONFIG, SINGLE_RUN_URL, translations } from './constants'
+import { FiltersActor } from './pageObjects/filters/filtersActor'
+
+test.describe('Single run page filters', () => {
+  let client: ApolloClient<NormalizedCacheObject>
+
+  test.beforeEach(() => {
+    client = getApolloClient()
+  })
+  // TODO: (ehoops) Add this actual test!
+  test('Annotation Author filter', async ({ page }) => {
+    const filtersPage = new FiltersPage(page)
+    await filtersPage.goTo('https://playwright.dev/')
+  })
+
+  test.describe('Object Name filter', () => {
+    test('should filter when selecting', async ({ page }) => {
+      const filtersPage = new FiltersPage(page)
+      const filtersActor = new FiltersActor(filtersPage)
```

Comment:
nit - wonder if we can have these 2 lines in the `beforeEach()` like with `client`